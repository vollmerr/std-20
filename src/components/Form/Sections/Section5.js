import React from 'react';
import { FieldRadioButtons } from 'state-template';

import schema from '../schema';

const Section5 = () => (
  <div className={'row'} role={'group'} aria-labelledby={'section5__header'}>
    <h3 id={'section5__header'} className={'col-md-12'}>5. Eligibility</h3>

    <FieldRadioButtons
      {...schema.eligibilityState}
      className={'col-md-6'}
      inline
    />

    <FieldRadioButtons
      {...schema.eligibilityNonState}
      className={'col-md-6'}
      inline
    />
  </div>
);

export default Section5;
