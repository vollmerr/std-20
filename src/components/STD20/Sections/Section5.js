import React from 'react';
import { A, FieldCheckboxes } from 'state-template';

import schema, { calNetLink } from '../schema';

const Section5 = () => (
  <fieldset className={'form-group'}>
    <div className={'row'}>
      <legend className={'col-md-12'}>5. Eligibility</legend>

      <FieldCheckboxes {...schema.eligibility} aria-describedby={'field__eligibility--help'} className={'col-md-6'} />

      <p id={'field__eligibility--help'} className={'col-md-6'}>
        <b>**</b>
        {' Local and State Government require a '}
        <A text={'Non-State Agency Service Policy and an Authorization Order (ATO)'} href={calNetLink} />
        {' to obtain eligibility prior to first order.'}
      </p>
    </div>
  </fieldset>
);

export default Section5;
