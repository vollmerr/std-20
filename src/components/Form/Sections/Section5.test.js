import React from 'react';
import { shallow } from 'enzyme';

import Section5 from './Section5';

const props = {};

let wrapper;
describe('Section5', () => {
  beforeEach(() => {
    wrapper = shallow(<Section5 {...props} />);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
