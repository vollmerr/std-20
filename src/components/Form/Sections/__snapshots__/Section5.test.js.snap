// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section5 should render correctly 1`] = `
<div
  aria-labelledby="section5__header"
  className="row"
  role="group"
>
  <div
    className="col-md-12"
    id="section5__header"
  >
    5. Eligibility
  </div>
  <Component
    className="col-md-6"
    helpId="eligibility--help-state"
    helpText="Complete Section 6. CATR/ATR Information below."
    inline={true}
    name="eligibility"
    options={
      Array [
        Object {
          "label": "State Government",
          "value": "state",
        },
      ]
    }
  />
  <Component
    className="col-md-6"
    helpId="eligibility--help-non-state"
    helpText={
      <React.Fragment>
         Requires a 
        <A
          href="https://cdt.ca.gov/services/calnet-ordering/"
          text="Non-State Agency Service Policy and an Authorization Order (ATO)"
          to=""
        />
         to obtain eligibility prior to first order.
      </React.Fragment>
    }
    inline={true}
    name="eligibility"
    options={
      Array [
        Object {
          "label": "Local Government",
          "value": "local",
        },
        Object {
          "label": "Federal Government",
          "value": "federal",
        },
      ]
    }
  />
</div>
`;
